lib/tasks/dump.rake

```diff
@@ -38,7 +38,7 @@ namespace :db do
     end
 
     desc 'Create anonymised database dump, compress (gzip) and upload to s3 - run on host via job (see run_job)'
-    task :anonymised => :environment do
+    task :anonymised, [:local] => :environment do |_task, args|
```

Comment:
This is (usually?) called from `kubernetes_deploy/scripts/job.sh` (by running `rake db:dump:run_job`) where an argument isn't provided.

If it is expected to be run directly is it worth adding something to `docs/development.md`?

Reply:
it would have defaulted to no args but, in any event, I have amended to infer localhost environment and not require the argument